Dockerfile

```diff
@@ -4,7 +4,7 @@ WORKDIR /app
 
 RUN apk update
 RUN apk upgrade --available
-RUN apk add libc6-compat openssl-dev build-base libpq-dev nodejs=~18 npm git python3
+RUN apk add libc6-compat openssl-dev build-base libpq-dev nodejs=~20 npm git python3
```

Comment:
I dont think this alpine docker image knows what node 20 is. https://pkgs.alpinelinux.org/packages?name=nodejs&branch=edge&repo=&arch=&maintainer=

we will need to bump the image being used. When I checked last Friday asdf nodejs plugin didn't have node 20 added to it either

Reply:
Yeah, v20 is too new so I'm going to leave this in draft

Reply:
asdf nodejs now has v20.9.0

Reply:
Alpine still using v18

Reply:
I've made a backlog card for updating it and added a note about Alpine not updating yet: https://trello.com/c/EIvl0xBy/1264-update-node-to-v209

There's a [Gitlab PR open for the update](https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/54105), I assume it'll be at least few days before that gets merged and released.

Reply:
@DavidBiddle node 20 should be available for alpine now

Reply:
Hmm still needs more looking at I think - getting this build failure:

```
> [build  5/16] RUN apk add libc6-compat openssl-dev build-base libpq-dev nodejs=~20 npm git python3:
0.547 ERROR: unable to select packages:
0.548   nodejs-current-20.8.1-r0:
0.548     breaks: world[nodejs~20]
0.548     satisfies: npm-9.6.6-r0[cmd:node]
```

Reply:
😭  so at the moment only alpine edge has nodejs 20.9........ alpine 3.18 hasn't been updated yet to support it.

Also  we will need to update the docker image being pulled in from `Dockerfile`